crates/rolldown_plugin/src/plugin_driver/build_hooks.rs

```diff
@@ -264,12 +279,23 @@ impl PluginDriver {
       if !self.plugin_usage_vec[plugin_idx].contains(HookUsage::Transform) {
         continue;
       }
+      let call_id = if tracing::enabled!(tracing::Level::TRACE) {
```

Comment:
I encountered an owneship problem here. To solve it, it requires refactoring the current code into a standalone function. So I go this way in the transform hook instead of injecting from context.